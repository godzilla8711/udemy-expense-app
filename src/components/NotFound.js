import React from 'react';
import ReactDOM from 'react-dom';

class NotFound extends React.Component {
  render() {
    return (
      <div>
        Page is not found.
      </div>
    );
  }
}

export default NotFound;
