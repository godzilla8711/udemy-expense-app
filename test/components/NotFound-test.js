import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../../src/components/NotFound';

describe('render', () => {
  it('renders component', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
