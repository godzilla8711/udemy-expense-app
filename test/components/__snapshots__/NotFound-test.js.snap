// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders component 1`] = `
<div>
  <p>
    The request page is not found.  Please click 
    <Link
      replace={false}
      to="/"
    >
      here
    </Link>
     to return to the Dashboard.
  </p>
</div>
`;
